### ISSUE TITLE
Validate submitting a COMPSs task through cimi-lifecycle, then retrieving its results

### ISSUE THREAD

**Issue by sstanovnik** [mF2C/mF2C#124]
Depends on #84  
Depends on #85
Depends on #86
Depends on #87 
Depends on #88

1) Submit a service that serves 1+1 30 seconds after start, trigger a job, validate that result-unavailable is returned, then try again after 30 seconds, validate that 2 is returned
2) Submit a service that raises a runtime exception, validate that an error is returned

**Comment by cjdcordeiro** [comment]
@sstanovnik is this still valid?

**Comment by sstanovnik** [comment]
It's valid and would be generally useful, but we're not planning to demonstrate this in the final demo for UC2, so it would only be done if the need arises. Don't know about the other usecases. I vote for close wontfix.
